docs/src/main/sphinx/release/release-405.md

```diff
@@ -0,0 +1,219 @@
+# Release 405 (21 Dec 2022)
+
+## General
+
+* Add support for setting an optional limit for the `query_max_run_time` session
+  property, which can be set using the `query.max-run-time.hard-limit`
+  configuration property. ({issue}`15115`)
+* Add Trino version to the outputs of `EXPLAIN` and `EXPLAIN ANALYZE`. ({issue}`15317`)
+* Add task input/output size distribution to the output of
+  `EXPLAIN ANALYZE VERBOSE`. ({issue}`15286`)
+* Add stage skewness warnings to the output of `EXPLAIN ANALYZE`. ({issue}`15286`)
+* Add support for configuring a refresh interval for the DB resource group
+  manager with the `resource-groups.refresh-interval` configuration property. ({issue}`14514`)
+* Improve performance of queries that compare `date` columns with
+  `timestamp(n) with time zone` literals. ({issue}`5798`)
+* Improve performance and resource utilization when inserting into tables. ({issue}`14718`, {issue}`14874`)
+* Improve performance for `INSERT` queries when fault-tolerant execution is
+  enabled. ({issue}`14735`)
+* Improve query planning performance of queries with many `GROUP BY` clauses. ({issue}`15292`)
+* Improve query performance for large clusters and skewed queries. ({issue}`15369`)
+* Improve performance of queries that read a small number of columns and queries
+  that process tables with large Parquet row groups or Orc stripes. Add the
```

Comment:
"ORC"

The comment about Parquet / ORC is connector-specific, so it should either be moved to the affected connectors or removed entirely.

Also, the last sentence doesn't provide any info on how it's related to this improvement. Generally, for optimizations we'd say "this optimization can be enable (or disabled) via the xxx configuration option or session property", but it's not clear in this case.